enterprise/backend/test/metabase_enterprise/serialization/api_test.clj

```diff
@@ -222,8 +222,6 @@
                                                       {:file ba}))
                           log (slurp (io/input-stream res))]
                       (testing "3 header lines, then cards+database+collection, then the error"
-                        (is (= #{"Card" "Database" "Collection"}
-                               (log-types (str/split-lines log))))
```

Comment:
It is non-deterministic whether the Dashboard gets serialized before the Collection fails.